README.md

```diff
@@ -210,6 +210,12 @@ custom:
 ```
 > Note that only relative path is supported at the moment.
 
+#### Usage with yarn
+
+Note that if auto-packing is enabled, the plugin will call `npm install`. If you are using yarn your `yarn.lock` file will be not be honored, which might lead to unexpected results as your dependencies will most likely not match (or be missing, as npm does not install packages in the same as npm).
```

Comment:
Shouldn't this be `[...] install packages in the same *way as yarn*`?

Reply:
Can't believe I did not see that, fixed!